skfda/exploratory/outliers/_directional_outlyingness_experiment_results.py

```diff
@@ -0,0 +1,93 @@
+# flake8: noqa
```

Comment:
A comment with the reference of the source of this data or the method to generate it would be useful.

Reply:
Added in 0e25f13d349922efd5355f84f2f3a05f947e7171.